app/src/main/java/com/jnj/vaccinetracker/reportsoverview/vaccinesoverview/screens/VaccinesOverviewFragment.kt

```diff
@@ -241,21 +258,27 @@ class VaccinesOverviewFragment : BaseFragment(),
         }
     }
 
+    @SuppressLint("SetTextI18n")
     private fun setupLocationsDropdown() {
         val locations = listOf(
             Constants.ALL_STRING,
             Constants.VISIT_PLACE_STATIC,
             Constants.VISIT_PLACE_OUTREACH,
             Constants.VISIT_PLACE_SCHOOL
         )
+
         val locationsAdapter = ArrayAdapter(requireContext(), R.layout.item_dropdown, locations)
         binding.autoCompleteLocations.setAdapter(locationsAdapter)
-
         binding.autoCompleteLocations.setText(Constants.ALL_STRING, false)
 
         binding.autoCompleteLocations.setOnItemClickListener { _, _, position, _ ->
             selectedLocation = locations[position]
             applyFilters()
+
+            if (selectedLocation != Constants.ALL_STRING) {
+                binding.autoCompleteVaccines.setText("")
```

Comment:
does it make sense to clear vaccines dropdown if any single location is set? can't location and vaccines be selected together?

Reply:
Resolved

Reply:
so what was your idea? maybe I miss something, I don't know your requirements